src/test-apps/happy/lib/WeaveNodeConfigure.py

```diff
@@ -331,7 +331,7 @@ def __init_node(self, node, node_params={}):
             self.logger.error("[localhost] WeaveNodeConfigure: %s" % emsg)
             return
 
-        if self.weave_state.getNodeType(node) in ['ap', 'service']:
+        if self.weave_state.getNodeType(node) in ['ap']:
             emsg = "ignoring initialization for nodes types 'ap' and 'service'"
```

Comment:
Comment and message no longer matches the code.

Reply:
good eyes:)
I will fix it